infrastructure/reftest/reftest_match-print-ref.html

```diff
@@ -0,0 +1,13 @@
+<title></title>
+<meta name="reftest-pages" content="1-2">
+<style>
+* {margin: 0, padding:0}
```

Comment:
Note that there's a comma here, is that valid as a CSS separator?

Reply:
It's not, good catch. But fixing it doesn't seem to change anything.